dvc/remote/base.py

```diff
@@ -86,6 +86,21 @@ def __repr__(self):
             path_info=self.odb.path_info or "No path",
         )
 
+    def __eq__(self, other):
+        return self.odb == other.odb
+
+    def __hash__(self):
+        return hash(self.odb)
+
+    @classmethod
+    def from_odb(cls, odb):
```

Comment:
This is a temporary workaround until the push/pull/save unification is completed (since we will no longer need Remote)